atd-vze/src/queries/recommendations.js

```diff
@@ -15,23 +15,9 @@ export const GET_RECOMMENDATION_LOOKUPS = gql`
 
 export const INSERT_RECOMMENDATION = gql`
   mutation InsertRecommendation(
-    $text: String
-    $update: String
-    $crashPk: Int
-    $userEmail: String
-    $recommendation_status_id: Int
-    $partner_id: Int
+    $recommendationRecord: recommendations_insert_input!
   ) {
-    insert_recommendations(
-      objects: {
-        rec_text: $text
-        rec_update: $update
-        crash_pk: $crashPk
-        created_by: $userEmail
-        recommendation_status_id: $recommendation_status_id
-        recommendations_partners: { data: { partner_id: $partner_id } }
-      }
-    ) {
+    insert_recommendations(objects: [$recommendationRecord]) {
```

Comment:
previously we had a bug where if you created a recommendation by just filling out the status, a recommendation record would be created along with a partner record that just had null as the partner id, so then you would end up with something that looks like this 
![image](https://github.com/user-attachments/assets/8e4424d8-529d-4c35-80b4-a9f6428ec7e9)
we dont want to create a partner record until the user actually adds a partner. so im changing this mutation to use an object variable that may or may not contain partner data so we arent creating empty partners